card/mod/01_core/spec/set/all/collection_spec.rb

```diff
@@ -52,7 +52,7 @@
       expect(c.contextual_content( context_card )).to eq("AlphaBeta")
     end
 
-    it "returns content even when context card is hard templated" do #why the heck is this good?  -efm
+    it "returns contesnt even when context card is hard templated" do #why the heck is this good?  -efm
```

Comment:
Line is too long. [103/80]<br>Missing space after #.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.